miette-derive/src/diagnostic_source.rs

```diff
@@ -59,7 +59,8 @@ impl DiagnosticSource {
                     };
                     quote! {
                         Self::#ident #display_pat => {
-                            std::option::Option::Some(#rel.as_ref())
+                            use std::borrow::Borrow;
+                            std::option::Option::Some(#rel.borrow())
```

Comment:
Can we do `std::option::Option::Some(std::borrow::Borrow::borrow(#rel))` instead?

Reply:
I meant could you use the entire qualified path, instead of the method form, to prevent ambiguity :)

Reply:
It's what I'm doing in this commit right? https://github.com/zkat/miette/pull/169/commits/417647574d7c9339ff739ee21cdd980de5ddde7e#diff-d856fc482d1e8e54cf0cfab0560bc32983812ba43914fb3cb9bf88325b1908a3R62

Or am I missing something?

Reply:
oh sorry, github displayed stuff funny. We're good here.